app/assets/stylesheets/dlme.scss

```diff
@@ -202,6 +206,11 @@ body {
   }
 }
 
+.image-masthead .background-container,
+.image-masthead .background-container-gradient {
+  top: 0;
```

Comment:
What is the purpose of this?

Reply:
Tested without it and it seemed to work, so I've removed this.